src/ui/colorui.js

```diff
@@ -59,15 +61,24 @@ export default class ColorUI extends Plugin {
 
 		/**
 		 * Label used by the dropdown.
+		 *
 		 * @type {String}
 		 */
 		this.dropdownLabel = dropdownLabel;
 
 		/**
 		 * The number of columns in the color grid.
+		 *
 		 * @type {Number}
 		 */
 		this.columns = editor.config.get( `${ this.componentName }.columns` );
+
+		/**
+		 * Keeps reference to {@link module:font/ui/colortableview~ColorTableView}.
+		 *
+		 * @type {module:font/ui/colortableview~ColorTableView}
+		 */
+		this.colorTableView;
```

Comment:
We're not declarations [like that](https://github.com/ckeditor/ckeditor5-font/blob/5f512d754067e61ed5c7b9d3e30030e3c739dfcd/src/ui/colorui.js#L81). Simply use @member api tag to declare a member. Like here: https://github.com/ckeditor/ckeditor5-font/blob/5f512d754067e61ed5c7b9d3e30030e3c739dfcd/src/fontcommand.js#L32

Reply:
I don't mind to change it.

However there is plenty of examples where we declare class properties in such way:
https://github.com/ckeditor/ckeditor5-engine/blob/master/src/model/document.js#L44-L96
or:
https://github.com/ckeditor/ckeditor5-font/blob/d4570220c7d7a0e1e8b5817f81c65bdf8be1a98e/src/ui/colorui.js#L75

`@type` seems to be nicer here, as you don't have possibility to make any typo mistake and you don't need to remember to update docs if parameter name will change.